src/main/java/com/adobe/acrobatsign/service/RestApiAgreements.java

```diff
@@ -849,4 +855,195 @@ public JSONObject workflowInfo(String accessToken, String workflowId) throws IOE
 		return responseJson;
 	}
 
+	public JSONObject getUsers(String accessToken) throws IOException{
+		// TODO Auto-generated method stub
+		
+		final String endpointUrl = getBaseURL() + GET_USERS;
+		final Map<String, String> headers = new HashMap<>();
+		headers.put(RestApiUtils.HttpHeaderField.AUTHORIZATION.toString(), accessToken);
+		
```

Comment:
no need for this method here, check the UserService.java. We already have a method related to users.